deploy/k8s/overlays/prod/kustomization.yml

```diff
@@ -0,0 +1,30 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: labelstudio
+
+namePrefix: labelstudio-
+
+bases:
+  - ../../base/
+
+resources:
+  - ingress.yml
+
+patchesStrategicMerge:
+  - env.yml
+
+commonLabels:
+  stage: production
+
+secretGenerator:
+- name: labelstudio
+  env: labelstudio.env
+
+images:
+  - name: postgres
+    newTag: '11.5'
+  - name: nginx
+    newTag: 'latest'
```

Comment:
Let's stick to the latest available version, eg `1.21`

Reply:
all right